crates/arrow_sql_gen/tests/clickhouse_test.rs

```diff
@@ -0,0 +1,73 @@
+use arrow_sql_gen::clickhouse::block_to_arrow;
```

Comment:
Let's add this to the bottom of `crates/arrow_sql_gen/src/clickhouse.rs` in a `mod tests`